pkg/continuoustest/util_test.go

```diff
@@ -141,6 +142,13 @@ func testVerifySamplesSumHistograms(t *testing.T, generateValue generateValueFun
 	// Round to millis since that's the precision of Prometheus timestamps.
 	now := time.UnixMilli(time.Now().UnixMilli()).UTC()
 
+	if strings.HasSuffix(testLabel, "histogram_int_gauge") {
+		// fixes flaky test for TestVerifySamplesSum/generateHistogramValue_for_histogram_int_gauge:should_return_error_if_there's_a_missing_series where the last histogram has an expected value of 0 so the number of series is irrelevant
```

Comment:
Can you clarify what this is doing any why? There's no need to mention that it's fixing a flaky test.

Reply:
Updated comment, hope that clarifies.